environments/development/base/vpc.tf

```
@@ -0,0 +1,20 @@
+#tfsec:ignore:aws-ec2-require-vpc-flow-logs-for-all-vpcs
+module "vpc" {
+  source = "github.com/trade-tariff/terraform-aws-vpc?ref=0ea859dd659701e6e8dda61e61c47629eeda5ba3"
+
+  name = "trade_tariff_development_vcp"
```

```suggestion
  name = "trade_tariff_development_vpc"
```

Tiny one but just for consistency, it's VPC 😄